app/server/lib/updateChecker.ts

```diff
@@ -48,7 +60,11 @@ export async function updateGristServerLatestVersion(gristServer: GristServer) {
   const latestVersionAvailable: LatestVersionAvailable = {
     version: response.latestVersion,
     isNewer: versions[1] !== installedVersion,
+    isCritical: response.isCritical ?? false,
+    dateChecked: Date.now(),
```

Comment:
This is a time on the user's server. I see a comparison made with it later, in the client's browser, is that right? Hmm, there's room for a little ugliness here, if clocks are a bit off, but it is all utc at least. Probably ok.